tests/testreactive.py

```diff
@@ -590,7 +590,7 @@ def gen():
 async def test_reactive_gen_pipe():
     def gen(val):
         yield val+1
-        time.sleep(0.05)
+        time.sleep(0.1)
```

Comment:
Avoid the flaky Windows test; use the same fix as https://github.com/holoviz/param/pull/967.